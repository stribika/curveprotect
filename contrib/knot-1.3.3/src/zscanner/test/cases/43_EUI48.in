$ORIGIN	.
$TTL	1

; OK
@	EUI48	00-00-00-00-00-00	; The simplest case
@	EUI48	FF-FF-FF-FF-FF-FF	; The maximal case
@	EUI48	aa-bb-cc-dd-ee-ff	; Lower-case
@	EUI48	\# 6 000000000000	; Hexadecimal rdata
@	TYPE108	\# 6 000000000000	; TYPE + Hexadecimal rdata
@	TYPE108	00-00-00-00-00-00	; TYPE
@	eui48	00-00-00-00-00-00	; Type in lower-case

; KO
@	EUI48
@	EUI48				; Empty rdata
@	EUI48	\# 0			; Hex empty rdata
@	EUI48	\#			; Missing hex length
@	EUI48	00-00-00-00-00		; Too few hex pairs
@	EUI48	00-00-00-00-00-00-00	; Too many hex pairs
@	EUI48	00-00-00-00-00-0	; Missing char in a hex pair
@	EUI48	00:00-00-00-00-00	; Bad separator
@	EUI48	00-00-00-x0-00-00	; Bad character
@	EUI48	00-00-00-00-00-00 x	; Unexpected item
@	EUI48	\# 7 000000000000 00	; Too long rdata
@	EUI48	\# 7 000000000000	; Bad rdata length
