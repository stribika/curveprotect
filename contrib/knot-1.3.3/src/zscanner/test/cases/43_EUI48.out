OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=FFFFFFFFFFFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=AABBCCDDEEFF
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=000000000000
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=006C
RDATA=000000000000
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_EBAD_EUI_LENGTH
------
WARNG=ZSCANNER_EBAD_EUI_LENGTH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_CHAR_DASH
------
WARNG=ZSCANNER_EBAD_HEX_CHAR
------
WARNG=ZSCANNER_EBAD_REST
------
WARNG=ZSCANNER_EBAD_HEX_RDATA
------
WARNG=ZSCANNER_EBAD_RDATA_LENGTH
------
